src/app/modules/members/components/members-list/members-list.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Subject } from 'rxjs';
+import { filter, takeUntil } from 'rxjs/operators';
+import { nonNull } from 'src/app/helpers/nonNull';
+import { selectCurrentCustomerOrganization } from 'src/app/store/current-customer/current-customer.selectors';
+import { getOrganizationMembers } from '../../store/members.actions';
+import { membersCount, selectMembers } from '../../store/members.selectors';
+import { FilterValues } from './members-list-filter-bar/filter-values.model';
+import { pageInfo } from './members-list.constants';
+
+@Component({
+  selector: 'vc-members-list',
+  templateUrl: './members-list.component.html',
+  styleUrls: [
+    './members-list.component.scss',
+  ],
+})
+export class MembersListComponent implements OnInit, OnDestroy {
+  curentCustomerOrganization$ = this.store.select(selectCurrentCustomerOrganization);
+
+  members$ = this.store.select(selectMembers);
+
+  totalCount$ = this.store.select(membersCount);
+
+  curentCustomerOrganizationId = '';
+
+  pageSize = pageInfo.pageSize;
+
+  currentPage = 1;
+
+  unsubscriber = new Subject();
+
+  constructor(private readonly store: Store) { }
+
+  onPageChange(newPage: number): void {
+    const after = (newPage - 1) * this.pageSize as unknown as string;
+    this.getMembers(this.pageSize, after);
+    this.currentPage = newPage;
+  }
+
+  onFilterChange(data: FilterValues): void {
+    console.log(data);
```

Comment:
Mark code like this with `// TODO` to prevent left it after changes in other stories. And yes, it's ok if Sonar will show warnings. Their disappearance after other stories completion will show what we do everything right.